src/components/anchor-navigation/anchor-navigation-test.stories.tsx

```diff
@@ -77,14 +83,17 @@ export const AnchorNavigationStory = () => {
     </AnchorNavigation>
   );
 };
+DefaultStory.storyName = "default";
 
-export const AnchorNavigationInFullScreenDialogStory = () => {
+export const InFullScreenDialogStory: ComponentStory<
```

Comment:
Something we need to consider, maybe not in this PR, but is to fix this: 
![Screenshot 2022-06-08 at 14 07 02](https://user-images.githubusercontent.com/56251247/172624038-07ce4745-4339-4be3-aa28-006fde826df4.png)

I think there are parameters in Storybook to specify that the code should be shown and not that ⬆️